src/test/java/org/folio/template/util/TemplateEngineHelperTest.java

```diff
@@ -40,4 +47,23 @@ void contextIsEnrichedWithAdditionalDateTimeTokens() {
     assertEquals(expectedJson, inputJson);
   }
 
-}
\ No newline at end of file
+  @Test
+  void mapToExceptionTest() {
+    Response response = TemplateEngineHelper.mapExceptionToResponse(new BadRequestException());
+    assertEquals(response.getStatus(), HttpStatus.SC_BAD_REQUEST);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+
+    response = TemplateEngineHelper.mapExceptionToResponse(new NotFoundException());
+    assertEquals(response.getStatus(), HttpStatus.SC_NOT_FOUND);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+
+    response = TemplateEngineHelper.mapExceptionToResponse(new InUseTemplateException());
+    assertEquals(response.getStatus(), HttpStatus.SC_BAD_REQUEST);
+    assertEquals(response.getMediaType(), MediaType.TEXT_PLAIN_TYPE);
+    assertEquals(response.getEntity(), "Cannot delete template which is currently in use");
+
+    response = TemplateEngineHelper.mapExceptionToResponse(new NullPointerException());
+    assertEquals(response.getStatus(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
+  }
+
+}
```

Comment:
There are two possibilities to assert for equality:
* `assertEquals(expected, actual)`
* `assertThat(actual, is(expected))` from org.hamcrest.MatcherAssert.assertThat

They differ in the order, the first one has `expected, actual`, the second `actual, expected`.

Please swap the parameters if you want to use `assertEquals`, or switch to `assertThat`.

Reply:
Fixed